content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md

```diff
@@ -0,0 +1,1545 @@
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+  <!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准的对象元数据。更多信息：
+  https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **spec** (<a href="{{< ref "../policy-resources/network-policy-v1#NetworkPolicySpec" >}}">NetworkPolicySpec</a>)
+  <!--
+  Specification of the desired behavior for this NetworkPolicy.
+  -->
+  spec 定义特定网络策略所需的所有信息.
+
+- **status** (<a href="{{< ref "../policy-resources/network-policy-v1#NetworkPolicyStatus" >}}">NetworkPolicyStatus</a>)
+  <!--
+  Status is the current state of the NetworkPolicy. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+  -->
+  status 是 NetworkPolicy 的当前状态。更多信息：
+  https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+
+## NetworkPolicySpec {#NetworkPolicySpec}
+
+<!--
+NetworkPolicySpec provides the specification of a NetworkPolicy
+-->
+NetworkPolicySpec 定义特定网络策略所需的所有信息.
+
+<hr>
+
+<!--
+- **podSelector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>), required
+
+  Selects the pods to which this NetworkPolicy object applies. The array of ingress rules is applied to any pods selected by this field. Multiple network policies can select the same set of pods. In this case, the ingress rules for each are combined additively. This field is NOT optional and follows standard label selector semantics. An empty podSelector matches all pods in this namespace.
+-->
+- **podSelector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>)， 必需
+  
+  podSelector 对该网络策略所适用的一组 Pod 进行选择。一组 ingress 入口策略将应用于此字段选择的任何 pods。多个网络策略可以选择同一组 pods。
+  在这种情况下，这些 ingress 列表的效果是相加的。 此字段不是可选的，并且遵循标准标签选择器语义。 
+  一个空值的 podSelector 匹配此命名空间中的所有 pods。
+
+<!--
+- **policyTypes** ([]string)
+
+  List of rule types that the NetworkPolicy relates to. Valid options are ["Ingress"], ["Egress"], or ["Ingress", "Egress"]. If this field is not specified, it will default based on the existence of Ingress or Egress rules; policies that contain an Egress section are assumed to affect Egress, and all policies (whether or not they contain an Ingress section) are assumed to affect Ingress. If you want to write an egress-only policy, you must explicitly specify policyTypes [ "Egress" ]. Likewise, if you want to write a policy that specifies that no egress is allowed, you must specify a policyTypes value that include "Egress" (since such a policy would not include an Egress section and would otherwise default to just [ "Ingress" ]). This field is beta-level in 1.8
+-->
+- **policyTypes** ([]string)
+
+  NetworkPolicy 相关的策略规则列表。 有效选项为 [“Ingress”]，[“Egress”] 或 [“Ingress”， “Egress”] 两者兼具。 
```

Comment:
@tengqm thanks fo all the reviewing and editing.  why do we use single quote here? the original english version doesn't use the single quote.

Reply:
Single quotes (backtiqs) are used for code style. The original English text is not good at expressing this because the text was generated from English comments written by developers. When generating docs, we'd struggle to avoid confusion and make things clearer.